src/modules/chat_replies/index.js

```diff
@@ -0,0 +1,21 @@
+const $ = require('jquery');
+const settings = require('../../settings');
+
+class ChatRepliesModule {
+    constructor() {
+        settings.add({
+            id: 'chatReplies',
+            name: 'Chat Replies',
```

Comment:
can we invert this setting name, so by default users have a false option to be enabled

Reply:
updated!